Teradata/vantage-ui-platform
# Fix/schematics ng add scss

closes #7 

## What's included?
Refactor ng-add @covalent/core so that

it doesn't add theme.scss anymore
put what is in theme.scss into bottom of styles.scss and remove theme.scss

### Test Steps

- Navigate to scripts directory
- Run ./test-schematics.sh : this is the script to test this entire feature locally
- this will create a folder `testxyz` in tmp directory.
- verify that teme.scss file should not be pressent in the testxyz/src folder.
- verfilt styles.scss should include theme.scss
